go/Makefile

```diff
@@ -25,9 +25,6 @@ lint:
 		| grep -v stutters \
 		| grep -v "error strings should not be capitalized" \
 		| grep -v prot.go \
-		| grep -v "redundant if ...; err != nil check, just return error instead" \
```

Comment:
this one is really mean

Reply:
i don't think that's in golint anymore...